examples/500-cli/cli.md

```diff
@@ -428,7 +427,7 @@ cat .warpforge/catalogs/my-catalog/warpsys.org/busybox/_mirrors.json
 	"catalogmirrors.v1": {
 		"byWare": {
 			"tar:4z9DCTxoKkStqXQRwtf9nimpfQQ36dbndDsAPCQgECfbXt3edanUrsVKCjE9TkX2v9": [
-				"file://.warpforge/warehouse/4z9/DCT/4z9DCTxoKkStqXQRwtf9nimpfQQ36dbndDsAPCQgECfbXt3edanUrsVKCjE9TkX2v9"
```

Comment:
Okay, so, I know we're plowing through some things that weren't getting exercised, so maybe there's been rot here -- but ideally, this stuff using local "file://" _should_ work, and we have vendored that file in the git repo for this purpose, and I think that if that works and the file is present, then hopefully these tests don't actually need network?

Reply:
hmm possibly. Not sure when/why/how that got switched over.

Reply:
Ah no that's entirely just the output of the command.
https://github.com/warptools/warpforge/pull/39/files/4f504b1c0a2a1a98b2e5886d01d4678acf6c536f#diff-8d8e5d1c040d3a0eab640b364644a15d331f15d74fd1d3ca0bd5115ac84f8623L399

We don't have a way to specify the location of the file without going into shell mode.

Reply:
We could feasibly _actually_ copy the warehouse into the workspace during the ExecFn instead of just setting an env var for it.

Reply:
(Maybe it should be WARPFORGE_FILE_WAREHOUSE_PREFIXPATH or something, on second thought, because maybe we wanted to explicitly have sone mirrors info file here?  I don't remember how explicit we were trying to be in this particular example / maybe worth briefly revisiting the intent.)

Reply:
We never did the `EXTRA_WAREHOUSE` thing. Getting this to work was generally a pain with paths, so that might be the way to go.

Reply:
What's the difference between those and `WARPFORGE_WAREHOUSE`?